pkg/generic/thrift/base.go

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2024 CloudWeGo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package thrift
+
+import "github.com/cloudwego/gopkg/protocol/thrift/base"
+
+// TrafficEnv ...
+// Deprecated: use github.com/cloudwego/gopkg/protocol/thrift/base
+type TrafficEnv = base.TrafficEnv
+
+// NewTrafficEnv ...
+// Deprecated: use github.com/cloudwego/gopkg/protocol/thrift/base
+func NewTrafficEnv() *TrafficEnv {
+	return base.NewTrafficEnv()
+}
+
+// Base ...
+// Deprecated: use github.com/cloudwego/gopkg/protocol/thrift/base
+type Base = base.Base
+
+// NewBase ...
+// Deprecated: use github.com/cloudwego/gopkg/protocol/thrift/base
+func NewBase() *Base {
```

Comment:
var NewBase = base.NewBase

Reply:
@jayantxie When defining like `var NewBase = base.NewBase`, the compiler ins't likely to do inlining because NewBase here is essentially a function pointer, so I'd like to write func like this.